app/src/main/java/com/example/fittracker/MainActivityModel.java

```diff
@@ -1,15 +1,19 @@
 package com.example.fittracker;
 
-public class MainActivityModel implements LogInContract.Model{
-    private UserBase listUsuarios;
-    public MainActivityModel (UserBase listUsuarios){
-        this.listUsuarios = listUsuarios;
+public class MainActivityModel implements LogInContract.Model {
+    private final String MAIL_PRUEBA = "danko94.cg@gmail.com";
+    private final String PASS_PRUEBA = "1234";
+    private UserBase lista;
+
+    public MainActivityModel() {
+        lista = new UserBase();
+        lista.addUsuario(new User(MAIL_PRUEBA, PASS_PRUEBA));
     }
 
     @Override
     public boolean isValidLogIn(String mail, String password) {
```

Comment:
you can pass here a User object with the mail and password entered by the user

Reply:
done